.travis.yml

```diff
@@ -3,8 +3,8 @@ os:
   - linux
   - osx
 julia:
-  - 0.7
   - 1.0
+  - 1.3
```

Comment:
better to say just `- 1` here, then when 1.4 comes out it will test on that instead of 1.3

Reply:
1.4 is now out so this line is definitely out of date

Reply:
Not as out of date as having a Require file! If you make a suggestion I'll click it, but I'm not sure anyone is watching.

Reply:
```suggestion
  - 1
```